gobblin-restli/gobblin-flow-config-service/gobblin-flow-config-service-api/src/main/pegasus/org/apache/gobblin/service/Issue.pdl

```diff
@@ -0,0 +1,46 @@
+namespace org.apache.gobblin.service
+
+/**
+ * Issue describes a specific unique problem in the job or application.
+ *
+ * Issue can be generated from log entries, health checks, and other places.
+ */
+record Issue {
+
+  /**
+   * Time when the issue have occured
+   */
+  time: Timestamp
```

Comment:
It's actually unclear what "long" means for Time. In the most common interpretation, unix timestamp is a number of seconds since 1970, but in our API it is milliseconds. Having a type reference with a description of what it means should be more clear. Timestamp type comment says that it is in milliseconds.

Personally, I think we should have used ISO format like "2015-04-14T11:07:36.639Z" that is both human- and computer- readable, but it's too late to change it since other parts of API use timestamps.  https://stackoverflow.com/a/29626123